pkg/util/password/dispatch.go

```
@@ -31,10 +31,18 @@ func resolveFormat(hash []byte) (passwordFormat, error) {
 }
 
 func Hash(password []byte) ([]byte, error) {
+	if len(password) > MaxLength {
+		password = password[:MaxLength]
+	}
```

Instead of truncating the password silently, can we return error?